routes/todo.go

```diff
@@ -0,0 +1,63 @@
+package routes
+
+import (
+	"errors"
+	"fmt"
+	"github.com/gofiber/fiber/v2"
+	"strconv"
+)
+
+func createErrorMap(err error) fiber.Map {
+	return fiber.Map{
+		"success": false,
+		"error":   err.Error(),
+	}
+}
+func (h handler) GetAll(c *fiber.Ctx) error {
```

Comment:
This code doesn't function correctly, when I make a request here on a new instance of the application it returns a success true even if there are no todos at all.
It shouldn't be the job of the frontend to validate the data from the backend (even if it's a good practice)

<img width="1497" alt="Screenshot 2023-07-08 at 14 12 09" src="https://github.com/kevinpita/go-todo/assets/84069271/a390a609-1b55-4e43-813a-61154aeecbcb">

Reply:
Fixed a24bfa701752e650fa687874c0f44ec07016468b